spark/build.sbt

```diff
@@ -0,0 +1,9 @@
+test in assembly := {}
+
+assemblyMergeStrategy in assembly := {
+  case "reference.conf" => MergeStrategy.concat
+  case "application.conf" => MergeStrategy.concat
+  case "META-INF/MANIFEST.MF" => MergeStrategy.discard
+  case "META-INF\\MANIFEST.MF" => MergeStrategy.discard
+  case _ => MergeStrategy.first
+}
```

Comment:
Spark subproject is the only one we ever build an assembly for, so it localized the settings (it was part of common settings before). Same story with the admin subproject and the `conflictWarning in ThisBuild := ConflictWarning.disable` setting.